openassessment/templates/openassessmentblock/oa_base.html

```diff
@@ -3,7 +3,7 @@
 <div class="wrapper wrapper--xblock wrapper--openassessment theme--basic">
     <div class="openassessment problem" id="openassessment">
         <div class="wrapper--grid">
-            <h2 class="openassessment__title problem__header">{{ title }}</h2>
+            <h2 class="openassessment__title problem__header">{% trans title %}</h2>
```

Comment:
@nedbat We were trying translate the default title name. @ssemenova tried to translate the default title where it is declared in openassessmentblock.py, but she ran into issues. Perhaps you have some advice-- I know that xblock translation support is pretty spotty.